axelrod/tests/unit/test_strategy_transformers.py

```diff
@@ -15,6 +15,20 @@ class TestClass(object):
 
 class TestTransformers(unittest.TestCase):
 
+    def test_helper_function_is_strategy_static(self):
+        self.assertTrue(is_strategy_static(axelrod.Cooperator))
+        self.assertFalse(is_strategy_static(axelrod.Alternator))
+
+    def test_helper_function_is_strategy_static_with_inherited_strategy(self):
+        class NewCooperator(axelrod.Cooperator):
+            pass
+
+        class NewAlternator(axelrod.Alternator):
+            pass
+
+        self.assertTrue(is_strategy_static(NewCooperator))
+        self.assertFalse(is_strategy_static(NewAlternator))
+
     def test_all_strategies(self):
         # Attempt to transform each strategy to ensure that implementation
```

Comment:
Woops my comment was actually a mistake because I misread the diff but if you've found a fail case that's good I guess although fingers crossed we can get a fix :confused: :)